### DISCUSSION TITLE
Template tag conflict

### DISCUSSION THREAD

**Discussion by phuhien92** [alpinejs/alpine#2408] [1. Help] [answered]
Is there any way to change <template/> tag to other tag like <script type=“text/x-template”/>. My app is currently using template tag for parsing server data, it will replace any template tag without exception.

**Comment by SimoTod** [comment, marked as answer]
Not possible at the minute because the content of a script tag is considered text and Alpine won't be able to find the root node. But maybe, if your conflict is server side, you can convert them to template tags on the client. Alpine mutation observer will automatically pick them up when you insert the updated node.

Useful links:
- https://developer.mozilla.org/en-US/docs/Web/API/Node/replaceChild (to swap tags) 
- https://developer.mozilla.org/en-US/docs/Web/API/Node/textContent (to read the script content) 
- https://developer.mozilla.org/en-US/docs/Web/API/Document/createElement (to create a template tag at runtime) 
- https://developer.mozilla.org/en-US/docs/Web/API/Element/innerHTML (to set the template content)
